suite/build.rs

```diff
@@ -0,0 +1,18 @@
+use std::env;
+use std::fs::File;
+use std::io::Write;
+use std::path::Path;
+
+fn main() {
+    let out_dir = env::var("OUT_DIR").expect("No out dir");
+    let dest_path = Path::new(&out_dir);
+    let mut f = File::create(&dest_path.join("memory.x")).expect("Could not create file");
+
+    f.write_all(include_bytes!("memory.x"))
+        .expect("Could not write file");
+
+    println!("cargo:rustc-link-search={}", dest_path.display());
+
+    println!("cargo:rerun-if-changed=memory.x");
+    println!("cargo:rerun-if-changed=build.rs");
+}
```

Comment:
Has this been tested?

Reply:
Can you clarify what you mean by tested?

This code is used every time the suite is compiled, to link the result using the memory.x file provided. 
Because the suite is correctly running on the qemu emulator, the linking must have succeeded.

Documentation for this can be found here: [riscv_rt Documentation](https://docs.rs/riscv-rt/latest/riscv_rt/)

Reply:
Makes sense.